client-app/core/api/graphql/push-messages/mutations/clearAllPushMessages/index.ts

```diff
@@ -9,7 +9,6 @@ export function useClearAllPushMessages() {
     optimisticResponse: {
       clearAllPushMessages: true,
     },
-    // TODO: Refactor updateQueries to use update since it will be deprecated in the next version of Apollo Client - https://www.apollographql.com/docs/react/api/react/hoc/#optionsupdatequeries
```

Comment:
The according task has been created: https://virtocommerce.atlassian.net/browse/VCST-1674